workflows/snakemake/Snakefile

```diff
@@ -0,0 +1,130 @@
+configfile: "./config.yaml"
+workdir: "/home/pkalverla1/Urban-M4/experiments/snakemake"
+envvars:
+    "NETCDF",
+
+rule DEFAULT:
+    # First rule is executed by default. It requires output of the last rule to be present.
+    input: "finished.clean"
+
+rule CREATE_WORKDIR:
+    input: workflow.source_path("namelist.wps"), workflow.source_path("GEOGRID.TBL"), workflow.source_path("namelist.input"), workflow.source_path("wrf.job")
+    output: "namelist.wps", "GEOGRID.TBL", "namelist.input"
+    shell: "cp {input} ."
+
+rule GEOGRID:
+    input: "namelist.wps", "GEOGRID.TBL"
+    output: "finished.geogrid"
+    shell:
+        """
+        {config[WPS_HOME]}/geogrid.exe
+        touch finished.geogrid
+        """
+
+rule W2W:
```

Comment:
I am getting this error when running this: 

`ImportError: libnetcdf-e68f7b33.so.19: cannot open shared object file: No such file or directory`

This might be due to how I set up the environment but I am not quite sure how to resolve it.